### ISSUE TITLE
Add check for VF METADATA that prevent an 'ital' axis if split

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#2828]
## Observed behaviour

Roboto (https://github.com/google/fonts/commit/c53c4df931d3fd0122a30b06d13126cfaad8e014) is a 'split' or discontinuous family - split roman and italic over two files, and I hand wrote METADATA.pb that was bad. 

## Expected behaviour

I should not have declared an `ital` axis

**Comment by felipesanches** [comment]
please, clarify:

The "bad" aspect of a `METADATA.pb` for such families is defined by having a declaration of an `ital` axis?

What's the reason for that? Is it that `ital` is actually boolean-on/off instead of a continuous axis?

If that's the case, then a `slant` axis would be OK, while an `ital` would be an offense?

**Comment by m4rc1e** [comment]
If I run `gftoolas add-font` on `ofl/roboto` it doesn't produce an ital axis.

We should have a general check that produces a FAIL if a METADATA.pb file contains an axis which isn't in the font.

**Comment by felipesanches** [comment]
yeah... maybe. But at this point I wouldn't add gftools as a FB dependency, as it has its own dependency tree hell :-P

**Comment by felipesanches** [comment]
Reading this again, I think I may have misunderstood @m4rc1e 's comment. I'll get back to it soon.
